python/cudf/cudf/io/orc.py

```diff
@@ -9,7 +9,9 @@
 import cudf
 from cudf import _lib as libcudf
 from cudf.utils import ioutils
-from cudf.utils.metadata import orc_column_statistics_pb2 as cs_pb2
+from cudf.utils.metadata import (  # type: ignore
+    orc_column_statistics_pb2 as cs_pb2,
+)
```

Comment:
Is there an option we could use to avoid having to add the `# type: ignore` comment?

Reply:
One solution is in `metadata/__init__.py` we have something like:

```python
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    orc_column_statistics_pb2 = object()
```

Is that preferable though?

Reply:
Wait actually, I wasn't thinking when I wrote the above comment. The following works (setup.cfg):

```
[mypy-cudf.utils.metadata.orc_column_statistics_pb2]
ignore_errors = True
```

Reply:
fixed

Reply:
Actually, we might just need the `type: ignore` here...

Reply:
The issue here is that `the `orc_column_statistics_pb2` module is built by `setup.py` while it looks like the style check is run without building anything. So our options are:

1. Build Python before running style checks
2. Ignore all typing errors in `orc.py`
3. Ignore typing errors in just this line (and anywhere else we may import `orc_column_statistics_pb2`)

(3) seems the most reasonable to me -  is that OK @kkraus14?

Reply:
3 seems the most reasonable to me for now and we can revisit in the future. Resolving this.